conf/cmi/core.entity_form_display.tpr_service.tpr_service.default.yml

```diff
@@ -80,7 +80,7 @@ content:
   field_metatags:
     weight: 26
     settings:
-      sidebar: true
```

Comment:
Why this is changed?

Reply:
This setting is applied after the the form is build (after form alter), so if it is set to true, we cannot manipulate the position of metatags field like it is done in node edit forms.